_layouts/post.html

```diff
@@ -47,7 +47,8 @@ <h5>Archive</h5>
         <section class="author">
           <h4> {{ author.display_name }} </h4>
           <p>
-            We are the Fast Ruby. Check us out at <a href="https://www.fastruby.io">www.fastruby.io</a>
+            I'm part of the Fast Ruby IO team. We offer a productized service to
```

Comment:
@etagwerker hm, looks a bit weird to me to put the `IO` there. What do you think?

Reply:
@lubc Just removed it.